### DISCUSSION TITLE
智慧城市》從醫療、交通到線上教學，不是單純比創新 關鍵是「以人為本」

### DISCUSSION THREAD

**Discussion by vvvioletta** [1101-GS4719-C/Python-Community-GS4719-C#83] [General]
今天來分享一則新聞——智慧城市。
之前比賽時曾經構想過類似的概念，只是我們當時是希望為合作企業打造智慧家庭生活圈（更符合企業盈利目標），但其實與智慧城市的構想是差不多的，利用IoT、5G以及AI來解決人們在日常生活中的痛點。
其中這篇新聞讓我最有心得的是：
1.我們中央所處的桃園竟然如此智慧！新聞裡面提到桃園用IoT等智慧系統來管理下水道和地下水的水資源，成功調度並且節水，每天還能支援新竹20萬噸用水，讓我十分訝異！由於現在實習的公司是面板製造業，對於用水十分重視（尤其是在寫ESG報告的時候），看到今年的各項評比都大幅進步，沒想到一大功臣竟是桃園市政府。
2.新聞中也有提到智慧交通產業，我第一個聯想到的是比賽時的智慧家庭生活圈。我們當時的主要目標即是利用IoT等技術結合智慧醫療，降低用戶的生病機率等等，假如能夠再結合智慧交通，便可以利用智慧巴士來接送老人來回醫院等服務，讓整個架構更完整。

看完這篇新聞突然覺得其實我們真的快要能夠生活在電影中的智慧城市中了，好振奮人心呀！
[https://www.wealth.com.tw/home/articles/34240](https://www.wealth.com.tw/home/articles/34240)

**Comment by chia-kai-chang** [comment]
URL連結的語法是這樣唷：

`[https://www.wealth.com.tw/home/articles/34240](https://www.wealth.com.tw/home/articles/34240)`

`[可以放置想要呈現的文字](實際URL連結)`
